bucket/deepstream.json

```diff
@@ -0,0 +1,28 @@
+{
+    "homepage": "https://deepstream.io",
+    "description": "The open realtime server, a fast and secure data-sync realtime server for mobile, web & iot",
+    "license": "AGPL-3.0-or-later",
+    "version": "4.2.5",
+    "architecture": {
+        "64bit": {
+            "url": "https://github.com/deepstreamIO/deepstream.io/releases/download/v4.2.5/deepstream.io-windows-4.2.5.zip",
+            "hash": "1f5bbfc92abb91795297a0ca7a5cfbf16c18cdfdef518c5265a45aece77de806"
+        }
+    },
+    "bin": "deepstream.exe",
+    "persist": "conf",
+    "env_set": {
+        "DEEPSTREAM_CONFIG_DIRECTORY": "$dir\\conf\\config.yml",
```

Comment:
Config directory and it points to file??

Reply:
> Depending on the installation sources, the configuration can be found in

> in the folder conf in the Node.js module directory if you installed with npm, or
> in the folder conf after unpacking the standalone server binary.

By default application is looking into this folder. Why this environment at all?

Reply:
About DEEPSTREAM_CONFIG_DIRECTORY, yes, it is strange.
If use a DIR, the error ocurrs: `Error: EISDIR: illegal operation on a directory, read`

Look this line: 
https://github.com/deepstreamIO/deepstream.io/blob/4dad5d7e2dde6adb1c095faba34506fe6e889961/src/config/js-yaml-loader.ts#L112
`verifyCustomConfigPath` check for a file.

> By default application is looking into this folder. Why this environment at all?

Check the current folder, instead of `%userprofile%\scoop\apps\deepstream\current`, so, if you change to `%userprofile%\scoop\apps\deepstream\current` and run `deepstream.exe`, works.

The value of `DEEPSTREAM_CONFIG_DIRECTORY` can be override with `-c or --config`

Reply:
https://github.com/deepstreamIO/deepstream.io/blob/4dad5d7e2dde6adb1c095faba34506fe6e889961/src/config/js-yaml-loader.ts#L127

```js
(config.libDir && fileUtils.lookupConfRequirePath(config.libDir)) ||
```

You do not have to have libdir environment if you have it inside config. Which is by default set to `../lib`.

![A](https://i.imgur.com/XzGuxIU.png)